src/card/aggregate-status/aggregate-status-card.html

```diff
@@ -29,7 +36,7 @@ <h2 class="card-pf-title">
   </div>
 </div>
 <div ng-if="$ctrl.isMiniLayout" class="card-pf card-pf-aggregate-status card-pf-aggregate-status-mini" ng-class="{'card-pf-accented': $ctrl.shouldShowTopBorder}">
-  <h2 class="card-pf-title">
+  <h2 class="card-pf-title" ng-class="{'hide-for-spinner': $ctrl.showSpinner}">
```

Comment:
Should we be hiding the title while data is loading?

Reply:
I believe the title is showing and not hidden with the base pfCard from my previous PR, I can change this to be consistent.